### PR TITLE
ci: shorten JRS tests used in PR Checks

### LINKED ISSUE #13605: Shorten JRS test run time used PR Checks
### Problem

Current JRS test used PR check took about 40 minutes to finish

### Solution

Shorten test time since the purpose of the test is to safe guard the test flow not test coverage, so
a shorter test should be enough to serve the purpose

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by JeffreyDallas**
**Description**:

Shorten test run time

**Related issue(s)**:

Fixes #13605 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: .github/workflows/node-flow-pull-request-checks.yaml [hunk 1]
```diff
@@ -251,33 +251,6 @@ jobs:
       gradle-cache-username: ${{ secrets.GRADLE_CACHE_USERNAME }}
       gradle-cache-password: ${{ secrets.GRADLE_CACHE_PASSWORD }}
 
-  abbreviated-panel:
```
Comment (Neeharika-Sompalli) [inline .github/workflows/node-flow-pull-request-checks.yaml:254]: Is this removing all tests from the Abbreviated panel @JeffreyDallas ? I thought we still need migration test as @cody-littley said

Comment (JeffreyDallas) [reply .github/workflows/node-flow-pull-request-checks.yaml:254]: This whole PR Abbrev test panel is kicked off every night

https://swirldslabs.slack.com/archives/C03F23RBDDF/p1717050550778969

so I wonder if we still need it for every PR check

Comment (cody-littley) [reply .github/workflows/node-flow-pull-request-checks.yaml:254]: Still useful to have the migration test for each PR

Comment (JeffreyDallas) [reply .github/workflows/node-flow-pull-request-checks.yaml:254]: do we want to keep the whole abrev panel for PC check,

```
	"configs/tests/FCM-Migration-5-2m.json",
	"configs/tests/Consistency-Hard-Restart-10m.json",
	"configs/tests/Consistency-NDReconnect-10m.json",
	"configs/tests/VMerkle-Basic-1k-4m.json"
```

if just keep the migration test (with a newer & shorter panel)

Comment (JeffreyDallas) [reply .github/workflows/node-flow-pull-request-checks.yaml:254]: Sorry, should be "PR Check"

**Comment by JeffreyDallas** [comment]
No longer needed

**Pull request closed without merging**